systemtests/tests/virtualfull-bscan/etc/bareos/bareos-dir.d/pool/Differential.conf

```diff
@@ -1,10 +1,13 @@
 Pool {
   Name = Differential
   Pool Type = Backup
-  Recycle = yes                       # Bareos can automatically recycle Volumes
-  AutoPrune = yes                     # Prune expired volumes
-  Volume Retention = 90 days          # How long should the Differential Backups be kept? (#09)
-  Maximum Volume Bytes = 10G          # Limit Volume size to something reasonable
-  Maximum Volumes = 100               # Limit number of Volumes in Pool
-  Label Format = "Differential-"      # Volumes will be labeled "Differential-<volume-id>"
+  Recycle = yes
+  AutoPrune = yes
+  Volume Retention = 90 days
+  Maximum Volume Bytes = 10G
+  Maximum Volumes = 100
+  Next Pool = VirtualFull
```

Comment:
Why add `Next Pool` here ? Is this file even needed ? I thought virtualfulls would use only incrementals.